administration/src/project-configs/koblenz/config.ts

```diff
@@ -1,8 +1,8 @@
 import BirthdayExtension from '../../cards/extensions/BirthdayExtension'
 import KoblenzReferenceNumberExtension from '../../cards/extensions/KoblenzReferenceNumberExtension'
+import { ActivationText } from '../common/ActivationText'
```

Comment:
Why does nuernberg not have such a file/uses this?

Reply:
there is no deeplink activation for nuernberg because they will only get a printed pdf